azbus/disposition.go

```diff
@@ -100,31 +101,9 @@ func (r *Receiver) DeadLetter(ctx context.Context, err error, msg *ReceivedMessa
 		azerr := fmt.Errorf("DeadLetter Message failure: %w", NewAzbusError(err1))
 		log.Infof("%s", azerr)
 	}
-	return nil
 }
 
-func (r *Receiver) Complete(ctx context.Context, msg *ReceivedMessage) error {
-	ctx = context.WithoutCancel(ctx)
-	log := r.log.FromContext(ctx)
-	defer log.Close()
-
-	span, _ := tracing.StartSpanFromContext(ctx, "Message.Complete")
-	defer span.Finish()
-
-	log.Infof("Complete Message")
-
-	err := r.receiver.CompleteMessage(ctx, msg, nil)
```

Comment:
why do we not r.receiver.CompleteMessage the message anymore?

Reply:
but we do - there were 2 versions of the complete() method and one has been deleted...